frappe/model/rename_doc.py

```diff
@@ -463,7 +463,7 @@ def get_link_fields(doctype: str) -> list[dict]:
 			.inner_join(dt)
 			.on(df.parent == dt.name)
 			.select(df.parent, df.fieldname, dt.issingle.as_("issingle"))
-			.where((df.options == doctype) & (df.fieldtype == "Link") & (df.is_virtual == 0))
+			.where((df.options == doctype) & (df.fieldtype == "Link") & (df.is_virtual == 0 or dt.is_virutal == 0))
```

Comment:
This should be `&`

Also Idk if OR even works in queries, did it work for you?

Reply:
Why should it be &? Either case when the doctype is virtual or the docfield is virtual should match. 

Yes 'OR' did work but '|' didn't.

Reply:
I meant you need to pull it out and use it as separate condition, like this:

```python
.where((df.options == doctype) & (df.fieldtype == "Link") & (df.is_virtual == 0) & (dt.is_virutal == 0))
```


doctype \ docfield | not virtual | virtual
-- | -- | --
not virtual | check | ignore
virtual | ignore | ignore

Reply:
Your current PR will break virtual doctypes where field isn't virtual. 

ALL virtual doctype fields are virtual, so devs don't actually set it to virtual on individual docfields.

Reply:
Ok, I see it now, thanks for explaining.